Sources/SwiftParser/Availability.swift

```diff
@@ -261,44 +261,30 @@ extension Parser {
   ///     platform-version → decimal-digits
   ///     platform-version → decimal-digits '.' decimal-digits
   ///     platform-version → decimal-digits '.' decimal-digits '.' decimal-digits
-  mutating func parseVersionTuple() -> RawVersionTupleSyntax {
+  mutating func parseVersionTuple(maxSubverNum: Int = 2) -> RawVersionTupleSyntax {
```

Comment:
I would call this maxComponentCount, which should IMO also include the major version component. I.e. `1.2.3` has a component count of 3 to me.